home4.0.10.c

```diff
@@ -0,0 +1,38 @@
+//4.0.10
+
+#include <stdio.h>
+#define N 10
+
+
+typedef struct{
+    float number[N];
+}City;
+

```

Comment:
it should be one struct. No sense for 2 structs here,...

Reply:
I fixed this task